### PR TITLE
Add basic svg of JBI flow and elements

### PR DESCRIPTION
**Pull request by bsieber-mozilla**
Basic diagram to discuss over RRA.

Can add additional diagrams, or additional elements to this diagram if desired.

### CODE REVIEW

**Review by kkleemola — APPROVED** [review]

**Comment by kkleemola** [comment]
:tada:

**Comment by grahamalama** [comment]
@bsieber-mozilla you might consider making this diagram with Mermaid so that changes can be more-easily tracked and [Github supports rendering the diagrams as images](https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/). But the diagram contents themselves look 💯.

**Comment by bsieber-mozilla** [comment]
@leplatrem @grahamalama what are your thoughts on the new updates?

File: diagrams/basic_jbi_diagram.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+``` mermaid
+graph TD
+    subgraph bugzilla services
+        A[Bugzilla] -.-|bugzilla event| B[(Webhook Queue)]
+        B --- C[Webhook Push Service]
+    end
+    D --> |create/update/delete issue| E[Jira]
+    D<-->|Read Bug| A
```
Comment (leplatrem) [inline diagrams/basic_jbi_diagram.md:8]: nitpick: only verb with uppercase

File: diagrams/basic_jbi_diagram.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    D -->|update see_also| A
+    subgraph jira-bugzilla-integration
+        C -.->|webhook event| D{JBI}
+        F["config.{ENV}.yaml"] ---|action configuration| D
```
Comment (leplatrem) [inline diagrams/basic_jbi_diagram.md:12]: nitpick: other arrows have verbs (eg. `read actions config`)

File: diagrams/basic_jbi_diagram.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+graph TD
+    subgraph bugzilla services
+        A[Bugzilla] -.-|bugzilla event| B[(Webhook Queue)]
+        B --- C[Webhook Push Service]
+    end
+    D --> |create/update/delete issue| E[Jira]
+    D<-->|Read Bug| A
+    D -->|update see_also| A
+    subgraph jira-bugzilla-integration
+        C -.->|webhook event| D{JBI}
```
Comment (leplatrem) [inline diagrams/basic_jbi_diagram.md:11]: Out of curiosity: why does this arrow have a dotted line? 
Also, shouldn't it have a verb too? (eg. `post /bugzilla_webhook` or `post webhook event`)

**Review by leplatrem — APPROVED** [review]
Much better 💯 

Since it is rendered in Github, shouldn't we put it in the README instead?

File: diagrams/basic_jbi_diagram.md (hunk 1 above)
Comment (bsieber-mozilla) [inline diagrams/basic_jbi_diagram.md:8]:
```suggestion
    D<-->|read bug| A
```

File: diagrams/basic_jbi_diagram.md (hunk 3 above)
Comment (bsieber-mozilla) [inline diagrams/basic_jbi_diagram.md:11]:
```suggestion
        C -.->|post /bugzilla_webhook| D{JBI}
```

File: diagrams/basic_jbi_diagram.md (hunk 2 above)
Comment (bsieber-mozilla) [inline diagrams/basic_jbi_diagram.md:12]:
```suggestion
        F["config.{ENV}.yaml"] ---| read actions config| D
```

File: diagrams/basic_jbi_diagram.md (hunk 3 above)
Comment (bsieber-mozilla) [reply diagrams/basic_jbi_diagram.md]: I used dotted lines for async operations; should I make them all solid lines?

File: README.md [hunk 4]
```diff
@@ -57,3 +57,21 @@ A bit more about the different fields...
 
 ### Custom Actions
 If you're looking for a unique capability for your team's data flow, you can add your own python methods and functionality[...read more here.](src/jbi/whiteboard_actions/README.md)
+
+
+### Diagram Oveview
```
Comment (leplatrem) [inline README.md:62]: typo: Overview

**Review by leplatrem — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (bsieber-mozilla) [inline README.md:62]:
```suggestion
## Diagram Overview
```

**Pull request merged**